alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -310,8 +310,17 @@ alonzoStyleWitness = do
          in seq (rnf ans) ans
   null unredeemed ?! UnRedeemableScripts unredeemed
 
-  {-  || { (sp, h) ∈ \fun{scriptsNeeded} utxo tx | h\mapsto s ∈ txscripts txw, s ∈ ScriptPhTwo } || = || fun{txrdmrs} tx ||  -}
-  let rdptrs = Set.fromList [el | (sp, _) <- sphs, SJust el <- [rdptr @era txbody sp]]
+  {-  ∀ (ptr \mapsto _ ) ∈ txrdmrs tx, ptr ∈ { rdptr txb sp | (sp, h) ∈ scriptsNeeded utxo tx,
+    h \mapsto s ∈ txscripts txw, s ∈ Scriptph2}  -}
```

Comment:
This comment is outdated, we want those sets to be equal (cf. the changes to the spec in this PR)